docs/yarn.lock

```diff
@@ -8952,9 +9650,9 @@ recursive-readdir@2.2.2:
     minimatch "3.0.4"
 
 redoc@^2.0.0-rc.31:
-  version "2.0.0-rc.33"
-  resolved "https://registry.yarnpkg.com/redoc/-/redoc-2.0.0-rc.33.tgz#df43f533bb0cc283cc209d69d2a91404a24bd8d1"
-  integrity sha512-1KLdnOU1aBIddgNBcEIU29h3VqXoTT493gT5hjyHg6sE91x9qEVWPYM2A+eETQFz5ygTwkBCp6xZDxVs+HIA9w==
+  version "2.0.0-rc.35"
```

Comment:
`make install-docs` failed on my machine with the following error:

```
warning "redoc > swagger2openapi > better-ajv-errors@0.6.7" has unmet peer dependency "ajv@4.11.8 - 6".
[4/4] 🔨  Building fresh packages...
[-/5] ⠈ waiting...
[2/5] ⠈ sharp
[-/5] ⠈ waiting...
[-/5] ⠈ waiting...
error /Users/max/Sources/rasa/docs/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments:
Directory: /Users/max/Sources/rasa/docs/node_modules/sharp
Output:
info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.7.4/libvips-8.7.4-darwin-x64.tar.gz
prebuild-install WARN install No prebuilt binaries found (target=13.11.0 runtime=node arch=x64 libc= platform=darwin)
gyp info it worked if it ends with ok
gyp info using node-gyp@5.0.7
gyp info using node@13.11.0 | darwin | x64
gyp info find Python using Python version 2.7.16 found at "/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python"
gyp info spawn /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
gyp info spawn args [
gyp info spawn args   '/usr/local/Cellar/node/13.11.0/libexec/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/max/Sources/rasa/docs/node_modules/sharp/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/usr/local/Cellar/node/13.11.0/libexec/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/max/Library/Caches/node-gyp/13.11.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/max/Library/Caches/node-gyp/13.11.0',
gyp info spawn args   '-Dnode_gyp_dir=/usr/local/Cellar/node/13.11.0/libexec/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/max/Library/Caches/node-gyp/13.11.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/max/Sources/rasa/docs/node_modules/sharp',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
```

any idea? should we pin the version of redoc to 33 for now?

Reply:
don't know what's happening, but did you do `yarn upgrade` or something?

Reply:
This looks like node-gyp errors, pretty common with packages like `sharp` that have big platform specific binaries in them. Install works fine for me. 
However you might want to change the `Makefile` to use `yarn install --frozen-lockfile`, so there's no patch version creep for contributors